Automattic/wp-calypso
# Performance Profiler: Update score section

## Proposed Changes

* Make the score bar bigger
* Make the score bar responsive in smaller screens
* Add maximum value
* Add status badge
* Add text with a link to recommendations


## Why are these changes being made?

To match the new proposed design

## Testing Instructions

* Go to `/speed-test-tool?url=wordpress.com`
* Check if the layout matches the ones below and figma
* Resize the screen to mobile sizes to guarantee the elements are wrapping correctly

| Desktop  | Mobile |
| ------------- | ------------- |
|![CleanShot 2024-09-05 at 19 16 12@2x](https://github.com/user-attachments/assets/1dafd427-2c69-4218-a39e-04d8a04721d6)|![CleanShot 2024-09-05 at 19 16 36@2x](https://github.com/user-attachments/assets/543267b1-ac42-4f97-86e9-ef42dfd447f7)|


## Pre-merge Checklist
